src/condor_schedd.V6/qmgmt.h

```diff
@@ -248,8 +248,8 @@ class JobQueueUserRec : public JobQueueBase {
 		int SchedulerJobsIdle=0;
 		int SchedulerJobsRunning=0;
 		int SchedulerJobsHeld=0;
-		int OCUClaims = 0;
 		int OCUJobsRunning = 0;
+		int OCUJobsWanted = 0; // Number of jobs that prefer an OCU claim, running or idle
```

Comment:
These counters appear to be leftover from a previous attempt?  one is never used at all, the other set but never referenced.